### ISSUE TITLE
Parametrize the execution of the WPK integration tests

### ISSUE THREAD

**Issue by DProvinciani** [wazuh/wazuh-qa#989]
### Description

As part of the epic https://github.com/wazuh/wazuh-qa/issues/979, we have to goal of stabilizing all the integration tests as well as document and organize the repository. Considering this, there is something that could be improved in the WPK integration tests.

The WPK integration tests are dependent on the Wazuh version so, for each new version of Wazuh, the tests have to be updated. This could be improved by parametrizing the execution of the tests as it is done with the FIM integration tests.

Basically, when the WPK integration tests are executed, we should have a way to specify the Wazuh version being tested and the test should take this version to run.

**Comment by DProvinciani** [comment]
Closing as invalid.
